src/utils/TypeUtils.ts

```diff
@@ -1,3 +1,7 @@
+import { SpokePoolClient } from "../clients";
+
 export type MakeOptional<Type, Key extends keyof Type> = Omit<Type, Key> & Partial<Pick<Type, Key>>;
 
 export type AnyObject = Record<string, unknown>;
+
+export type SpokePoolClients = Record<string, SpokePoolClient>;
```

Comment:
This is normally specified as `{ [chainId: number]: SpokePoolClient }`, but afaik object keys are coaxed into being strings anyway. Does it make a difference whether it's `Record<string, ...>` or `Record<number, ...>` ?

Reply:
I totally agree. This is something that I keep seeing and is why I decided to just go ahead and collapse it. Let me run through the code and verify that I've replaced all the references to spokeChainClients.

As a general rule, it's not that painful to switch between each, but it will require a loop so we should be uniform. Should we do `Record<number, SpokePoolClient>` everywhere?

Reply:
Changed all of the outstanding `{ [chainId: string | number]: SpokePoolClient }` into `SpokePoolClients`